system/argtable3/CMakeLists.txt

```diff
@@ -82,6 +82,11 @@ if(CONFIG_SYSTEM_ARGTABLE3)
   target_sources(argtable3 PRIVATE ${SRCS})
 
   if(NOT EXISTS ${CMAKE_BINARY_DIR}/apps/include/argtable3.h)
+
+    if(NOT EXISTS ${CMAKE_BINARY_DIR}/apps/include)
```

Comment:
should we create `${CMAKE_BINARY_DIR}/apps/include` by default? @xuxin930

Reply:
Good question, wanted to have minimal changes here.
Will wait for @xuxin930 response - or if I will discover any other default configuration relying on build folder include directory I will make it invariant.

Reply:
Done as xuxin930 suggested.